### DISCUSSION TITLE
Jupyter issue???

### DISCUSSION THREAD

**Discussion by kunstgarrison** [cooperrc/computational-mechanics#229] [General]
Was working on HW and was kicked off of Jupyter...

Anyone else having the same issue?

![Screenshot 2024-01-04 212838](https://github.com/cooperrc/computational-mechanics/assets/154924404/a4e91377-78ec-40a2-bf71-a050a2b11bad)

**Comment by jadongs** [comment]
I am having the same problem as well. :(

**Comment by riley-chamberlin** [comment]
Yes I think everyone is having the same problem

**Comment by XanderRaymond** [comment]
I keep getting on for a few minutes and kicked off.

**Reply by XanderRaymond** [reply]
Ok I can not get in at all anymore.

**Comment by jackseitz** [comment]
Yeah guess it's down, hopefully it'll be back up by tomorrow

**Comment by cooperrc** [comment]
Sorry about the hassle, not sure what happened. 

I don't see any issues when I log in today, 

https://compmech.uconn.edu

**Reply by owenlotito** [reply]
It is pretty touch-and-go still for me. I would hate to lose hours of work because it doesn't save or if it crashes.

**Reply by cooperrc** [reply]
I can restart the server, but it will interrupt everyone. 

Do it?

**Reply by owenlotito** [reply]
It actually is working better now, no need to restart server.
